ompi/mca/mtl/psm2/help-mtl-psm2.txt

```diff
@@ -45,3 +45,7 @@ Unknown path record query mechanism %s. Supported mechanisms are %s.
 #
 [message too big]
 Message size %llu bigger than supported by PSM2 API. Max = %llu
+#
+[no psm2 cuda env]
+Using CUDA enabled OpenMPI but PSM2_CUDA environment variable is %s.
```

Comment:
Open MPI is two words, not one.

Also, please use full sentences and good grammar in these messages.  These are meant to be formal help messages displayed to the user, not "good enough" messages that are only intended for developers.  Also, the way you are adding an additional lengthy string via the call to `opal_show_help()` is... unconventional.

More below, where you actually call `opal_show_help()`.